app/httpPresenter.js

```diff
@@ -14,6 +14,7 @@ class HttpPresenter {
     }
 
     presentFailure(error) {
+        // error.httpCode = undefined in case something went wrong in code and i get typeError or other errors
         this.response.status(error.httpCode).send({ message: error.message });
```

Comment:
If u catch an error, u should convert it into API error (when needed, and it is needed, if u pass it into presenter)
Common server error code is 500, u can use it along such cases